packages/@aws-cdk/aws-rds/lib/cluster-ref.ts

```diff
@@ -50,9 +50,16 @@ export interface DatabaseClusterAttributes {
   readonly port: number;
 
   /**
-   * The security group for this database cluster
+   * The security group identifier of the database cluster
+   *
+   * @deprecated use `securityGroup` instead
    */
-  readonly securityGroupId: string;
+  readonly securityGroupId?: string;
+
+  /**
+   * The security group of the database cluster
+   */
```

Comment:
since rds is still experimental, we should just remove `securityGroupId` and make `securityGroup` required.